src/main/java/mat/server/service/impl/MeasurePackageServiceImpl.java

```diff
@@ -122,7 +125,8 @@
     @Autowired
     private CompositeMeasurePackageValidator compositeMeasurePackageValidator;
 
-    private String currentReleaseVersion;
+    @Value("${mat.measure.current.release.version}")
+    private String currentReleaseVersion = "v6.0";
```

Comment:
What happens when you assign a value and use the value annotation?

Reply:
it overrides what was there from the property. need to remove the string.